orchestration/dagster_orchestration/hca_orchestration/tests/test_resources.py

```diff
@@ -57,3 +57,38 @@ class DataflowBeamRunnerTestCase(unittest.TestCase):
     def test_resource_can_be_initialized(self):
         with initialize_resource(dataflow_beam_runner) as dataflow_runner:
             self.assertIsInstance(dataflow_runner, DataflowBeamRunner)
+
+
+@solid(required_resource_keys={"load_tag"})
+def load_tag_solid(context) -> str:
+    tag: str = context.resources.load_tag
+    return tag
+
+
+class LoadTagTestCase(unittest.TestCase):
+    def test_load_tag_with_suffix(self):
+        result = execute_solid(
+            load_tag_solid,
+            run_config={
+                "resources": {"load_tag": {"config": {
+                    "load_tag_prefix": "fake_prefix",
+                    "append_timestamp": True
+                }}}
+            },
+            mode_def=ModeDefinition(resource_defs={"load_tag": load_tag})
+        )
+        self.assertTrue(result.output_value().startswith("fake_prefix"))
+
+    def test_load_tag_no_suffix(self):
+        result = execute_solid(
+            load_tag_solid,
+            run_config={
+                "resources": {"load_tag": {"config": {
+                    "load_tag_prefix": "fake_prefix",
+                    "append_timestamp": False
+                }}}
+            },
+            mode_def=ModeDefinition(resource_defs={"load_tag": load_tag})
+        )
+
+        self.assertEqual(result.output_value(), "fake_prefix")
```

Comment:
testing a resource by affixing it to a solid seems really roundabout. why can't we just test it by initializing it with `initialize_resource` and asserting things about its properties/methods?

Reply:
Gah, yeah you're right, and there is a perfectly usable pattern right above it. Fixing.